cmd/terramate/cli/cli.go

```diff
@@ -1120,9 +1130,9 @@ func (c *cli) eval() {
 			fatal(err, "eval %q", exprStr)
 		}
 
-		var out []byte
+		var o []byte
```

Comment:
`i` ok, `j` ok, `s` ok, `v` ok, but `o` ? haha unusual choice but I'm actually fine with it =)

Reply:
I was in "fixing build errors really fast" mode so I just reduced out to o :laughing:, but can change =P